sources/models/src/modeled_types/mod.rs

```diff
@@ -62,6 +62,21 @@ pub mod error {
             field: String,
             source: serde_plain::Error,
         },
+
+        #[snafu(display("Invalid Kubernetes Quantity '{}'", input))]
+        InvalidKubernetesQuantity { input: String },
+
+        #[snafu(display("Failed to parse {} as f32: {}", input, source))]
+        ParseToF32 {
+            input: String,
+            source: std::num::ParseFloatError,
+         },
```

Comment:
We're only using this for when it's a invalid threshold percentage value. So let's make the error reflect that.
```suggestion
        #[snafu(display("Invalid percentage value '{}': {}", input, source))]
        InvalidPercentage {
            input: String,
            source: std::num::ParseFloatError,
         },
```